contracts/src/v0.8/shared/test/enumerable/EnumerableMapAddresses.t.sol

```diff
@@ -75,63 +97,89 @@ contract EnumerableMapAddresses_contains is EnumerableMapAddressesTest {
     assertTrue(s_addressToAddressMap.set(address(this), address(this)));
-    assertTrue(s_addressToAddressMap.length() == 1);
+    assertEq(s_addressToAddressMap.length(), 1);
     assertTrue(s_addressToAddressMap.remove(address(this)));
-    assertTrue(s_addressToAddressMap.length() == 0);
+    assertEq(s_addressToAddressMap.length(), 0);
   }
 
   function testBytes32LengthSuccess() public {
-    assertTrue(s_addressToBytes32Map.length() == 0);
+    assertEq(s_addressToBytes32Map.length(), 0);
     assertTrue(s_addressToBytes32Map.set(address(this), MOCK_BYTES32_VALUE));
-    assertTrue(s_addressToBytes32Map.length() == 1);
+    assertEq(s_addressToBytes32Map.length(), 1);
     assertTrue(s_addressToBytes32Map.remove(address(this)));
-    assertTrue(s_addressToBytes32Map.length() == 0);
+    assertEq(s_addressToBytes32Map.length(), 0);
+  }
+
+  function testBytesLengthSuccess() public {
+    assertEq(s_addressToBytesMap.length(), 0);
+    assertTrue(s_addressToBytesMap.set(address(this), MOCK_BYTES_VALUE));
+    assertEq(s_addressToBytesMap.length(), 1);
+    assertTrue(s_addressToBytesMap.remove(address(this)));
+    assertEq(s_addressToBytesMap.length(), 0);
   }
 }
 
 contract EnumerableMapAddresses_at is EnumerableMapAddressesTest {
   using EnumerableMapAddresses for EnumerableMapAddresses.AddressToBytes32Map;
   using EnumerableMapAddresses for EnumerableMapAddresses.AddressToAddressMap;
+  using EnumerableMapAddresses for EnumerableMapAddresses.AddressToBytesMap;
 
   function testAtSuccess() public {
-    assertTrue(s_addressToAddressMap.length() == 0);
+    assertEq(s_addressToAddressMap.length(), 0);
     assertTrue(s_addressToAddressMap.set(address(this), address(this)));
-    assertTrue(s_addressToAddressMap.length() == 1);
+    assertEq(s_addressToAddressMap.length(), 1);
     (address key, address value) = s_addressToAddressMap.at(0);
-    assertTrue(key == address(this));
-    assertTrue(value == address(this));
+    assertEq(key, address(this));
+    assertEq(value, address(this));
   }
 
   function testBytes32AtSuccess() public {
-    assertTrue(s_addressToBytes32Map.length() == 0);
+    assertEq(s_addressToBytes32Map.length(), 0);
     assertTrue(s_addressToBytes32Map.set(address(this), MOCK_BYTES32_VALUE));
-    assertTrue(s_addressToBytes32Map.length() == 1);
+    assertEq(s_addressToBytes32Map.length(), 1);
     (address key, bytes32 value) = s_addressToBytes32Map.at(0);
-    assertTrue(key == address(this));
-    assertTrue(value == MOCK_BYTES32_VALUE);
+    assertEq(key, address(this));
+    assertEq(value, MOCK_BYTES32_VALUE);
+  }
+
+  function testBytesAtSuccess() public {
+    assertEq(s_addressToBytesMap.length(), 0);
+    assertTrue(s_addressToBytesMap.set(address(this), MOCK_BYTES_VALUE));
+    assertEq(s_addressToBytesMap.length(), 1);
+    (address key, bytes memory value) = s_addressToBytesMap.at(0);
+    assertEq(key, address(this));
+    assertEq(keccak256(value), keccak256(MOCK_BYTES_VALUE));
```

Comment:
suggestion: no need for `keccak`, you can compare directly. Foundry implements this function:

> function assertEq(bytes memory left, bytes memory right) internal;

```suggestion
    assertEq(value,MOCK_BYTES_VALUE);
```